enterprise-suite/values.yaml

```diff
@@ -90,6 +90,12 @@ exposeServices: false
 # If exposeServices is set to "NodePort" or "LoadBalancer", Console will be exposed on this port.
 esConsoleExposePort: 30080
 
+# Default monitors ConfigMap. Must contain file `default-monitors.json`.
+defaultMonitorsConfigMap: console-api-default-monitors
+
+# Static rules ConfigMap. Must contain file `static-rules.yml`.
+staticRulesConfigMap: console-api-static-rules
```

Comment:
Would it make sense to organize it a bit more hierarchically? I think these are the first console-api specific parameters.

```
consoleAPI:
   defaultMonitorsConfigMap:
   staticRulesConfigMap:
```

Reply:
Not feeling too strongly on this one - but it does feel like our parameters are all over the place at the moment.

Reply:
It would be nice, though all other values that we have are top-level and not hierarchical...

Reply:
Anyway, I grouped them. We should start to group other values into a hierarchy too whenever we make helm chart changes.